### PR TITLE
Home: Add new top-level documentation landing page, indexing the main entrypoints

### PR DESCRIPTION
**Pull request by amotl**
## About
There was a proposal to have a dedicated landing page for the documentation section at https://cratedb.com/docs/, which should refer to the most relevant 1st-level pillars of our documentation tree.

**Preview:** [Docs Home](https://cratedb-guide--94.org.readthedocs.build/home/)

## Details
This page is special, should not necessarily be part of the content tree, that's why it is flagged as **orphaned**, concerning Sphinx technicalities.

The aim is to slot this page into the URL namespace as a landing page for all the documentation based on Sphinx, straight at https://cratedb.com/docs/, by using corresponding Nginx configuration snippets.

## References
- https://github.com/crate/tech-content/issues/105

## Thoughts
This is just a quick proposal, which most definitively has flaws on different details. It intends to give you a first idea how this page could look like, mostly using the most venerable visual design element we are currently using across the board, a **card** component, applied in multiple variations, because otherwise it would be too boring.

Please suggest both wording improvements, and also please have your voice on general structure and layout, in terms of guidance, and how you would like to see the document being re-shaped. Please take your time, or, alternatively, if you roughly agree, acknowledge early, in order to postpone further editing to subsequent iterations, by shipping early, and iterating progressively.

## Contributions
We will be extraordinarily happy to receive your feedback on this PR, or at https://github.com/crate/tech-content/issues/105, when applicable. If you intend to work on another variant of this page hands-on, please derive a new feature branch from `collab/welcome`, and submit your patches creating a separate PR, which will also give you a dedicated preview rendering on RTD.

If you want to suggest different icons for the relevant items, please head over to https://fonts.google.com/icons.

/cc @seut, @matriv, @proddata, @simonprickett, @surister, @karynzv, @hlcianfagna, @hammerhead, @msbt, @matkuliak, @ckurze, @stephanec76, @geragray

### CODE REVIEW

**Comment by matriv** [comment]
I like this layout, I think  it's good to have a top level page that links to all (or the most important aspects of the whole eco-system (core db, cloud, drivers, examples, tutorials,etc.) I think that the community portal should be also visible on the right side, maybe even higher (before the drivers)?

**Comment by proddata** [comment]
As discussed in slack some suggested change in the latest commit.

File: docs/home/index.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+:::
+
+::::
+
+
+::::{grid} 1 1 2 2
+:margin: 1
+:padding: 2
+
+:::{grid-item-card} {material-outlined}`auto_stories;1.7em` Reference Manual
```
Comment (matkuliak) [inline docs/home/index.md:50]: Probably a pasting missclick, this points to cloud

File: docs/home/index.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+:link: cloud-docs-index
```
Comment (matkuliak) [inline docs/home/index.md:51]: Probably a pasting missclick, this points to cloud

Comment (amotl) [reply docs/home/index.md]: Thanks for spotting. Fixed!

File: docs/home/index.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+:padding: 2
+:class-title: sd-fs-5
+
+Want to dive deep into CrateDB? Explore the comprehensive reference manual.
+
+At the core of CrateDB Cloud sits CrateDB, the open-source, distributed, and
+scalable SQL database system designed for storing and analyzing massive amounts
+of data in real-time, even with complex queries.
+
+Based on Lucene, and inheriting from Elasticsearch/OpenSearch, CrateDB supports
```
Comment (proddata) [inline docs/home/index.md:62]:
```suggestion
Based on Lucene and inherited from Elasticsearch/OpenSearch, CrateDB supports
```
consistency in verb tense / smoother reading

File: docs/home/index.md [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+:link-type: ref
+:link-alt: Ecosystem Catalog
+:padding: 3
+:class-card: sd-pt-3
+:class-title: sd-fs-5
+:class-body: sd-text-center
+:class-footer: text-smaller
+{material-outlined}`category;3.5em`
++++
+Discover integrations and solutions from the open source community and CrateDB partners.
```
Comment (proddata) [inline docs/home/index.md:210]:
```suggestion
Discover integrations and solutions from the open-source community and CrateDB partners.
```

**Review by proddata — APPROVED** [review]

**Comment by proddata** [comment]
Let's get it live 🚀

**Pull request merged**